courses/free-courses-ko.md

```diff
@@ -236,7 +239,18 @@
 
 ### Spring
 
-* [스프링 입문 - 코드로 배우는 스프링 부트, 웹 MVC, DB 접근 기술](https://www.inflearn.com/course/%EC%8A%A4%ED%94%84%EB%A7%81-%EC%9E%85%EB%AC%B8-%EC%8A%A4%ED%94%84%EB%A7%81%EB%B6%80%ED%8A%B8) (김영한)
+* [스프링 입문 - 코드로 배우는 스프링 부트, 웹 MVC, DB 접근 기술](https://www.inflearn.com/course/%EC%8A%A4%ED%94%84%EB%A7%81-%EC%9E%85%EB%AC%B8-%EC%8A%A4%ED%94%84%EB%A7%81%EB%B6%80%ED%8A%B8) - 김영한 (Inflearn)
+
+
+### Svelte
```

Comment:
I don't think having two separate categories for svelte and svelte kit is helpfil. Categories are there to help folks find things. In the english lists, svelte is a subcategory under javascript, not sure why it should be different in the korean list.

Reply:
Thanks for the feedback. I think I forgot that Svelte is a web framework like React and created a separate category for it.... I'll fix it with a JavaScript subcategory.